src/Compositor.cpp

```diff
@@ -206,8 +206,7 @@ void CCompositor::setRandomSplash() {
 
 static std::vector<SP<Aquamarine::IOutput>> pendingOutputs;
 
-//
-void CCompositor::initServer() {
+void                                        CCompositor::initServer(std::string socketName, int socketFd) {
```

Comment:
and this is why the `//` was above this :) please add it back

Reply:
??????????????????????

Reply:
:)